API.html

```diff
@@ -1267,6 +1267,12 @@
     Boolean flag for optionally expanding output with additional whitespace
     for readability. Results in larger but more readable GeoJSON responses.
     </dd>
+    <dt>empty_fields</dt>
+    <dd>
+    Default is <samp>true</samp>.
+    <br>
+    Boolean flag for optionally skipping null properties.
```

Comment:
I'd also suggest using the same wording as in the parameter name (empty vs null). Might confuse some people.